packages/react/src/components/Storage/StorageManager/hooks/useUploadFiles/useUploadFiles.ts

```diff
@@ -85,7 +85,7 @@ export function useUploadFiles({
             progressCallback: onProgress,
             errorCallback: onError,
             provider,
-          }) as unknown as UploadTask; // @todo: fix type returned from uploadFile
+          }) as unknown as UploadTask;
```

Comment:
Do we still need the cast to unknown here?

Reply:
Unfortunately, I couldn't get uploadFile to return UploadTask due to how Amplify JS Storage types work...